### ISSUE TITLE
Which datasets to start with?

### ISSUE THREAD

**Issue by mankoff** [cryo-data/discuss#4]
Consider this a meta-discussion about how to have this discussion.

I'm not sure the best way to make this decision. GitHub allows "voting" by selecting emojis. For example:

👍🏿 BedMachine
😆  MeASURES 0478 https://nsidc.org/data/nsidc-0478
🎉  PROMICE Greenland Total Mass Balance https://doi.org/10.22008/FK2/OHI23Z

And then you can select the emoji(s) you agree with and the votes are tallied at the bottom of this comment box.

Another way would be to build a small table, and then you should be able to *edit this comment* (rather than replying in the text box below) and add a number if you agree or disagree, add a row if you want a new dataset, etc.

| Name  | DOI | Comment | Vote|
| ------------- | ------------- |-----------|-----------|
| BedMachine | https://doi.org/10.1002/2017GL074954 | No comment | kdm|
| MeASURES 0478 | https://nsidc.org/data/nsidc-0478 | Ice Velocity | kdm |
| PROMICE Greenland Total Mass Balance | https://doi.org/10.22008/FK2/OHI23Z | Updates daily | kdm
| PROMICE automatic weather station data |  https://doi.org/10.22008/promice/data/aws | No comment | aw
| Calving Front Machine (CALFIN) | https://doi.org/10.7280/D1FH5D | No comment | aw

Or we can just reply in text format and have a discussion that way...

**Comment by wehrad** [comment]
I can not edit your comment at the moment, could you update the permissions? Thanks!

**Comment by wehrad** [comment]
What should we do with data providers such as the Copernicus Data Store (CDS; e.g. giving access to  ERA5) but only through API/website requests? (hence not pointing directly to a DOI) 

We should probably stick to datasets with direct DOIs only, but I think this is an interesting discussion!

**Comment by mankoff** [comment]
> I can not edit your comment at the moment, could you update the permissions? Thanks!

I've changed base permissions to `write`. Does this fix it?

**Comment by wehrad** [comment]
> > I can not edit your comment at the moment, could you update the permissions? Thanks!
> 
> I've changed base permissions to `write`. Does this fix it?

Yes, thank you!

**Comment by mankoff** [comment]
> What should we do with data providers such as the Copernicus Data Store (CDS; e.g. giving access to ERA5) but only through API/website requests? (hence not pointing directly to a DOI)
> 
> We should probably stick to datasets with direct DOIs only, but I think this is an interesting discussion!

I do not think we need to stick to DOId data. But we should (for now) stick to easy-to-access data, that is fetchable by the datalad scripts. If we have to write our own custom scripts then `datalad fetch` doesn't work, and everything becomes much more complicated for the end-user. 

NSIDC did not support datalad access, but I believe that their support is important enough that I opened an issue with datalad to support NSIDC, which led to NSIDC changing their authentication method. See https://github.com/datalad/datalad/issues/5846

**Comment by jasonebox** [comment]
am interested in a SICE discussion, and an ESSD draft is in preps, hoping that GitHub is as useful as e.g. Discord for not only having a conversation but just as important, being able to find content later!

**Comment by twilamoon-science** [comment]
I think while this activity is happening as a GRISO WG activity, it's important to focus on Greenland data. And might consider also starting with data held in Greenland, Denmark, or NSF Arctic Data Center. Since QGreenland is working to make geospatial data more accessible, perhaps this tool makes sense to start with things that are more challenging in that space - e.g., complete weather station data with full time series.
